cmd/subnetcmd/delete.go

```diff
@@ -48,12 +50,13 @@ func deleteSubnet(cmd *cobra.Command, args []string) error {
 	// but only if no other subnet is using it.
 	// More info: https://github.com/ava-labs/avalanche-cli/issues/246
 
-	if _, err := os.Stat(subnetDir); err == nil {
-		// exists
-		os.RemoveAll(subnetDir)
-	} else {
+	if _, err := os.Stat(subnetDir); err != nil {
```

Comment:
We will want to review all of these `Stat`s for the above issue.

Reply:
Same as above -- totally up to you guys to decide what the right thing to do is in cases like this one.

Would you mind handling such cases? You'd know better than me whether we want to ignore or propagate errors in various places.